doc/changelog.rst

```diff
@@ -19,11 +19,15 @@ To be released at some future point in time
 
 Description
 
+- Expose first_device parameter for scripts, functions, models
 - Added support for MINBATCHTIMEOUT in model execution
 - Remove support for RedisAI 1.2.5, use RedisAI 1.2.7 commit
 
 Detailed Notes
 
+- Added support for first_device parameter for scripts, functions,
+  and models. This causes them to be loaded to the first num_devices
+  beginning with first_device (PR389_)
```

Comment:
We may want to merge `develop` and check that your entries are correctly added above Julia's.